lib/ansible/modules/storage/netapp/na_ontap_gather_facts.py

```diff
@@ -46,6 +46,13 @@
             - use "help" to get a list of supported facts for your system.
         default: "all"
         version_added: 2.8
+    max_records:
+        description:
+            - Limits maximum amount of records returned per query.
+                It must be between 1 and 4294967295.
+        default: "1024"
```

Comment:
I wouldn’t use quotes

Reply:
right, this is misleading. i will correct it here